packages/validate/src/index.ts

```diff
@@ -27,15 +28,20 @@ const validateOptions: ValidateOptions = {
 
 async function getLocalSchemaPaths(localSchemasDir: string, options: ValidateOptions): Promise<string[]> {
   const { ignoredPaths = [], ignoredExts = [] } = options;
-  const directories = (await readDir(localSchemasDir))
+  const dirsAndFiles = (await readDir(localSchemasDir))
     .filter((name: string) => !ignoredPaths.includes(name) && ignoredExts.every((ext) => !name.endsWith(ext)))
     .map((dir: string) => path.resolve(localSchemasDir, dir)) as string[];
```

Comment:
Could you check if that casting `as string[]` is useful by removing it ?
I suspect it is not.